tests/installation/module_registration/view_development_versions.pm

```diff
@@ -1,16 +1,17 @@
 # SUSE's openQA tests
 #
-# Copyright 2021 SUSE LLC
+# Copyright 2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
-# Summary: View list of available extension and modules including
-# development versions in module registration.
+# Summary: Uncheck 'hide development versions'
```

Comment:
the current description was more readable. "View" is more general, 'check', 'tick' 'untick' all these stuff is better for another layer.